helm_lib/templates/_csi_controller.tpl

```diff
@@ -8,6 +8,11 @@
   {{- $provisionerTimeout := $config.provisionerTimeout | default "600s" }}
   {{- $attacherTimeout := $config.attacherTimeout | default "600s" }}
   {{- $resizerTimeout := $config.resizerTimeout | default "600s" }}
+  {{- $snapshotterTimeout := $config.snapshotterTimeout | default "600s" }}
+  {{- $provisionerWorkers := $config.provisionerWorkers | default "10" }}
```

Comment:
It's up to consider, the default value 100, but I found that not every previsioner is capable to handle that, so 10